src/kbsb/national_elo_archive/api_national_elo_archive.py

```
@@ -0,0 +1,275 @@
+def get_pool():
+    global _pool
+    if _pool is not None:
+        return _pool
+    try:
+        pg_config = get_secret("postgres")
+        db_host = pg_config.get('dbhost', '')
+        if db_host == "oldelo.zerotwo.cloud":
+            try:
+                import socket
+                socket.gethostbyname(db_host)
+            except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)